spec/features/orders/show_spec.rb

```diff
@@ -0,0 +1,85 @@
+require "rails_helper"
```

Comment:
I think this is in the wrong test file? should be in /spec/users/orders/show_spec.rb